### PR TITLE
Couch to SQL migration: CommtrackConfig (3 of 3)

### PR DESCRIPTION
**Pull request by orangejenny**
Branched off of https://github.com/dimagi/commcare-hq/pull/27598, first new commit is marked PR3

### CODE REVIEW

**Review by millerdev — APPROVED** [review]

File: corehq/apps/commtrack/models.py [hunk 1]
```diff
@@ -56,6 +56,9 @@ def set_actions(self, actions):
         self.actionconfig_set.set(actions, bulk=False)
         self.set_actionconfig_order([a.id for a in actions])
 
+    def action_by_keyword(self, keyword):
```
Comment (orangejenny) [inline corehq/apps/commtrack/models.py:59]: This was accidentally removed in d8604dee38570e15f0084c3006118ae63a474680

File: corehq/apps/commtrack/models.py [hunk 2]
```diff
@@ -56,6 +56,9 @@ def set_actions(self, actions):
...
+        return dict((a.keyword.lower(), a) for a in self.all_actions).get(keyword.lower())
```
Comment (millerdev) [inline corehq/apps/commtrack/models.py:60]: nit
```suggestion
        return {a.keyword.lower(): a for a in self.all_actions).get(keyword.lower()}
```

**Review by millerdev — APPROVED** [review]

File: corehq/apps/commtrack/models.py [hunk 3]
```diff
@@ -51,47 +46,18 @@ class SQLCommtrackConfig(SyncSQLToCouchMixin, models.Model):
...
-                field: getattr(sql_submodel, field)
-                for field in spec['fields']
-            })
-            sql_submodel.save()
-        couch_object.save(sync_to_sql=False)
-
-    @classmethod
-    def _migration_get_couch_model_class(cls):
-        return CommtrackConfig
+        return {a.keyword.lower(): a for a in self.all_actions}.get(keyword.lower()}
```
[the remark this answers is not in the document]
Comment (millerdev) [reply corehq/apps/commtrack/models.py:60]: oof, sorry about that

**Review by millerdev — APPROVED** [review]

**Comment by orangejenny** [comment]
Removed migration label, which was added when this was branched off of another PR. Merging.

**Comment by esoergel** [comment]
I'm excited to see [this graph](https://app.datadoghq.com/dashboard/jmg-mcq-cy8?from_ts=1619471552536&fullscreen_section=overview&fullscreen_widget=2986558727980624&live=true&to_ts=1622063552536&fullscreen_start_ts=1619471587778&fullscreen_end_ts=1622063587778&fullscreen_paused=false) drop!

**Pull request merged**